conda-recipe/vcdat/meta.yaml

```diff
@@ -20,4 +20,4 @@ source:
   git_rev: v0.0.1
```

Comment:
@SandiSamuel we should change this to master  or . some new tag point since we changed the version tag